src/jni_driver.cpp

```diff
@@ -33,6 +34,23 @@ JNIEnv* jniEnv() {
 	return env;
 }
 
+int AndroidUtils::openDocument(const char* path, const char* perms) {
+    auto env = jniEnv();
+
+    auto clazz = env->FindClass("com/panda3ds/pandroid/AlberDriver");
+    auto method = env->GetStaticMethodID(clazz, "openDocument", "(Ljava/lang/String;Ljava/lang/String;)I");
```

Comment:
Maybe not in this PR, but I recommend that you collect method ids once during startup so you're not finding classes each time you want to open a document. It can become pretty costly.

Reply:
Implemented, thanks!